src/main/java/seedu/parser/IndexParser.java

```diff
@@ -37,12 +36,12 @@ public static int getIndexFromInput(String userInput)
             return REMOVE_ALL_ID;
         }
         if (!significantIndex.matches(numbers)) {
-            throw new MissingIndexException();
+            throw new NumberFormatException();
```

Comment:
User input of index is not in valid regex therefore NumberFormatException. No issue here.